apps/aefate/src/aefa_fate_op.erl

```diff
@@ -2976,3 +2984,34 @@ gt_to_emcl(X1, X2, X3, X4, X5, X6, X7, X8, X9, X10, X11, X12) ->
     emcl:mk_Gt(emcl:mk_Fp(X1), emcl:mk_Fp(X2), emcl:mk_Fp(X3), emcl:mk_Fp(X4),
                emcl:mk_Fp(X5), emcl:mk_Fp(X6), emcl:mk_Fp(X7), emcl:mk_Fp(X8),
                emcl:mk_Fp(X9), emcl:mk_Fp(X10), emcl:mk_Fp(X11), emcl:mk_Fp(X12)).
+
+-ifdef(DEBUG_INFO).
+dbg_loc({immediate, File}, {immediate, Line}, EngineState) ->
+    Info  = aefa_debug:set_debugger_location({File, Line}, aefa_engine_state:debug_info(EngineState)),
+    Stack = aefa_engine_state:call_stack(EngineState),
+    UpdatedStatus = 
+    case lists:member({File, Line}, aefa_debug:breakpoints(Info)) of
+        true  ->
+            case aefa_debug:debugger_status(Info) of
+                continue -> aefa_debug:set_debugger_status(break, Info);
```

Comment:
I have checked the Erlang debugger, and it does not break neither on stepover (called next) nor on stepout (called finish). That's why the breakpoints will be hit only with continue, or stepin (which executes every line anyway, so there's no need to mention it here).